test/test_performance.py.in

```diff
@@ -65,21 +71,16 @@ def _raw_to_png(csv_data, png_path):
                'latency_mean (ms)',
                'latency_variance (ms) * 100',
                'maxrss (Mb)']].plot(x='T_experiment', secondary_y=['maxrss (Mb)'])
-
-    plt.title('@TEST_NAME@ Performance Test')
-
+    plt.title('Performance tests latency\n@TEST_NAME@')
     plt.savefig(png_path)
 
     dataframe[['T_experiment', 'cpu_usage (%)']].plot(x='T_experiment')
-
-    plt.title('@TEST_NAME@ CPU usage (%) Test')
-
+    plt.title('Performance tests CPU usage (%)\n@TEST_NAME@')
+    plt.ylim(0, 100)
     plt.savefig(png_path[:-4] + "_cpu_usage.png")
 
     dataframe.plot(kind='bar',y=['received', 'sent', 'lost'])
-
-    plt.title('@TEST_NAME@  Received/Sent packets per second and Lost packets')
-
+    plt.title('Performance tests Received/Sent/Lost packages\n@TEST_NAME@')
```

Comment:
You should change all of the other references to packages if you're going to change it here.